Workbooks/ADXCluster/AtResourceTables/ResourceTableDetails.workbook

```diff
@@ -897,6 +897,18 @@
       },
       "name": "On Boarding Message Group-Tables"
     },
+    {
+      "type": 1,
+      "content": {
+        "json": "<img src=\"https://raw.githubusercontent.com/microsoft/Application-Insights-Workbooks/master/Documentation/Images/ADXClusterTablesSample.png\" alt=\"Sample image is unavailable\" style=\"width:1250px;\"/>"
+      },
+      "conditionalVisibility": {
+        "parameterName": "TableLogs",
+        "comparison": "isEqualTo",
+        "value": "not present"
+      },
+      "name": "sample text"
```

Comment:
it's the name of the image component right? I think it better to be called "sample image"

Reply:
same for other PRs